tests/php/BackupCode/VerifyHandlerTest.php

```diff
@@ -17,6 +18,18 @@ class VerifyHandlerTest extends SapphireTest
 {
     protected static $fixture_file = 'VerifyHandlerTest.yml';
 
+    public function setUp()
+    {
+        parent::setUp();
+
+        //Ensure that the handler for notifications is Email - which is stubbed with Swift_NullTransport in test state
+        Injector::inst()->load([
+            Email::class . '.MFA.Notification' => [
```

Comment:
Does this actually work in practice? I notice that you create the service definition in this test, but it's not defined in practice. Does the class that loads `Email.MFA.Notification` actually work without it being defined?

Reply:
Yes, it works as intended :)

Reply:
Intended meaning it falls back up to `...Email.MFA` when `...Email.MFA.NonExistentService` doesn't exist?

Reply:
Yup, exactly :)

Reply:
This code is removed.